openreview/venue_request/venue_request.py

```diff
@@ -1179,7 +1179,7 @@ def setup_request_form(self):
             },            
             'Submission Start Date': {
                 'description': 'When would you (ideally) like to have your OpenReview submission portal opened? Please specify the date and time in GMT using the following format: YYYY/MM/DD HH:MM(e.g. 2019/01/31 23:59). (Skip this if only requesting paper matching service)',
-                'value-regex': '.*',
+                'value-regex': r'^[0-9]{4}\/([1-9]|0[1-9]|1[0-2])\/([1-9]|0[1-9]|[1-2][0-9]|3[0-1])(\s+)?((2[0-3]|[01][0-9]|[0-9]):[0-5][0-9])?(\s+)?$',
```

Comment:
Nice! thanks for making the fix, could you add a test in one of the existing test files posting a venue request with an invalid date and expecting to get an error?